versioning/scripts/dockerfiles/main.go

```diff
@@ -45,6 +48,21 @@ func verifyDockerfiles(spec versions.Spec, tmpl template.Template) {
 		}
 	}
 
+	err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
+		check(err)
+		if info.Name() == "Dockerfile" && !info.IsDir() && !foundDockerfile[path] {
+			log.Printf("%s: UNIDENTIFIED (only WARNING)", path)
+		}
+		return nil
+	})
+	check(err)
+
+	if failureCount == 0 {
+		log.Print(">>> Summary: PASSED")
```

Comment:
Also consider adding a warning counter, so if warning counter is non zero, we'll output:
`log.Print("Dockerfile verification completed: PASSED (with warnings)")`